README.md

```diff
@@ -36,6 +36,23 @@ for s := range singers {
 }
 ```
 
+### Connection URL Properties
+
+The Cloud Spanner GORM supports the following connection URL properties
+
+#### Commonly Used Properties
+- credentials (String): File name for the credentials to use. The connection will use the default credentials of the environment if no credentials file is specified in the connection string. Example: `projects/my-project/instances/my-instance/databases/my-db;credentials=/path/to/credentials.json`
+- usePlainText (boolean): : Boolean that indicates whether the connection should use plain text communication or not. Set this to true to connect to local mock servers that do not use SSL. Example: `projects/test-project/instances/test-instance/databases/test-db;usePlainText=true`
```

Comment:
I don't think this is a commonly used property for end users. That is; I assume that they can connect to the emulator using the `SPANNER_EMULATOR_HOST` environment variable. That would otherwise be the most common use of this property.